nixos/doc/manual/development/writing-nixos-tests.section.md

```diff
@@ -21,10 +21,13 @@ import ./make-test-python.nix {
 }
 ```
 
-The attribute `testScript` is a bit of Python code that executes the
+We refer to the whole test above as a test module, whereas the values
+in [`nodes.<name>`](#opt-nodes) are NixOS modules. (A NixOS configuration is a module.)
```

Comment:
```suggestion
We refer to the whole test above as a test module, whereas the values
in [`nodes.<name>`](#opt-nodes) are NixOS modules themselves.
```

This should just link to a section that defines modules, once we have it. For now let's assume readers know what a NixOS module is, so we don't have to change this bit later.